### DISCUSSION TITLE
Recurring extra rewards for voting

### DISCUSSION THREAD

**Discussion by e101stella** [Zrips/CMI#3357] [Feature Requests]
**Description of feature/improvement request:**
It would be great if it could be made so that in the config you can set it up so that people get a reward for every X amount of votes. 

---
### Detailed explanation of what it should do, when and in what situation: ###
It should be an option that you can specify in the config, that allows it to setup a recurring reward. Ideally it could be a separate section in extra rewards.

### Basic explanation as to why this feature is needed and/or useful to have: ###
I want to incentivise voting for the server on vote sites, and one of the ways we do this is by giving a big reward for every 100 votes and this would enable us to automate the process.

**Comment by HeartlessAUS** [comment]
this here would really be a great feature, apologies for the necro, but i didnt want  to make a new threas about it
